code/index.html

```diff
@@ -2,17 +2,83 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <title>Kreativhang</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
```

Comment:
Don't forget the meta tag ;)
`<meta name="viewport" content="width=device-width, initial-scale=1">`